index.html

```diff
@@ -9,15 +9,16 @@
 
     <link rel="preconnect" href="https://fonts.googleapis.com">
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
-    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap" rel="stylesheet">  
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&display=swap" rel="stylesheet">  
 </head>
 <body>
+    <div class="blank"></div>
```

Comment:
- [Optional] Kindly consider removing the space at the top of your page so that your design can look exactly like the figma template. 
![1](https://user-images.githubusercontent.com/94551484/192059317-a2ddb161-9528-4384-8e43-73ef6c42cc33.png)